### PR TITLE
Capability changes

### PR DESCRIPTION
**Pull request by colugo**
I've made a lot of editorial changes to make the capability model more complete, and appropriate for the handbook.

I've also held off on adding in the assessment / hr component for now, which I can add back in after some wider DTO conversations about strategic hiring.

### CODE REVIEW

File: tmp/.htmlproofer/cache.log [hunk 1]
```diff
@@ -0,0 +1 @@
...ndbook/discovery/3-finishing-discovery/3-3-digital-service-standard.html"],"status":200,"message":""},"https://www.dto.gov.au/standard/design-guides/user-research/user-stories":{"time":"2016-02-26 09:05:08 +1100","filenames":["./_site/service-handbook/discovery/3-finishing-discovery/3-3-digital-service-standard.html"],"status":200,"message":""},"http://www.smashingmagazine.com/2010/06/design-better-faster-with-rapid-prototyping":{"time":"2016-02-26 09:05:08 +1100","filenames":["./_site/service-handbook/capability/3-prototyping/index.html"],"status":200,"message":""},"https://www.dto.gov.au/standard/design-guides/user-research/scenarios":{"time":"2016-02-26 09:05:08 +1100","filenames":["./_site/service-handbook/discovery/3-finishing-discovery/3-3-digital-service-standard.html"],"status":200,"message":""},"https://heroku.com":{"time":"2016-02-26 09:05:08 +1100","filenames":["./_site/service-handbook/alpha/3-building-prototypes/3-2-tools-technology.html"],"status":200,"message":""},"https://www.flickr.com/photos/gdsteam/20351061738":{"time":"2016-02-26 09:05:08 +1100","filenames":["./_site/service-handbook/discovery/2-starting-discovery/2-5-artefacts.html"],"status":200,"message":""}}
```
Comment (joshprice) [inline tmp/.htmlproofer/cache.log:1]: might want to add `/tmp` to `.gitignore`

File: _capability/3-prototyping/index.md [hunk 2]
```diff
@@ -24,27 +24,49 @@ These engineers build models to find out whether their designs will work. Aerosp
...
+ - Either by technical or license limitations, the prototype is only available to people situated with the development team.
 
-### Level 2
- - Designers are using the production development tools (such as HTML and CSS) to produce working prototypes
- - Prototypes are published to a 'live' environment, and deployed continuously throughout their development
- - The prototypes are used to supplement user research, with activities such as alpha-beta testing used to gather large amounts of supporting data  
+
+### Moving to level 1
+
+ - Paper prototypes are more useful then static WYSIWYG produced mockups. They take less time to produce, are easy to modify and can be distributed easily.
```
Comment (joshprice) [inline _capability/3-prototyping/index.md]: s/then/than/

File: _capability/3-prototyping/index.md [hunk 3]
```diff
@@ -24,27 +24,49 @@ These engineers build models to find out whether their designs will work. Aerosp
...
+ - Start to share your prototypes with a wider audience of users. This may mean that you need to rethink how you explain the flow of your prototypes, and what distribution mechanisms you use.
+ - Focus on the high-value parts of the prototype, and skip everything else. Large amounts of text or content that doesn't help to evaluate this approach to the MVP will probably distract people.
```
Comment (joshprice) [inline _capability/3-prototyping/index.md:45]: Large amounts of text will probably distract users

File: _capability/3-prototyping/index.md [hunk 4]
```diff
@@ -24,27 +24,49 @@ These engineers build models to find out whether their designs will work. Aerosp
...
+ - Read up on [Sketching in Code](http://alistapart.com/article/sketchingincode), which gets you started on developing prototypes in the technologies used for the final product.
+
+# Level 1
+
+Prototypes are being used successfully during your service design process, and the team collaborate on the designs before they are given to developers to implement.
```
Comment (joshprice) [inline _capability/3-prototyping/index.md:50]: collaborate_s_

File: _capability/3-prototyping/index.md [hunk 5]
```diff
@@ -24,27 +24,49 @@ These engineers build models to find out whether their designs will work. Aerosp
...
+### Indications that you're operating at level 1
+
+ - Designs are first tested on paper with the team and users before becoming digital. The paper designs go up on the wall for the team to refer to throughout the process.
+ - The designers and developers collaborate on the prototypes, ensuring that all aspects of the design are implementable and technically sound
+ - Developers implement the paper prototypes simple CSS, HTML and JavaScript, which introduces navigation and have just enough interactive components to demonstrate the particularly interesting functionality.
+ - Although they're throwaways, the prototypes are treated as important team technical assets and are stored in the team's version control repository.
+
+### Moving to level 2
+
+ - Start pair-programming with your designers when implementing coded prototypes. [There](http://www.w3schools.com/html/default.asp) are [heaps](http://www.w3schools.com/css/default.asp) of free learning resources online to help people learn (or re-learn) how to build content for the web.
```
Comment (joshprice) [inline _capability/3-prototyping/index.md:61]: while w3schools seems to have finally update their site, I think there are better resources

File: _capability/4-development/index.md [hunk 6]
```diff
@@ -13,42 +13,53 @@ Development converts ideas, assumptions, and user stories into a running, functi
...
 > String upn;
 
- - Code is needlessly complex, difficult to read and poorly formatted
- - There are large sections of commented out code
- - The code doesn't reflect the user's vocabulary:
+- Even though it works well and passes every test, the code is needlessly complex, difficult to read and poorly formatted
+- There are large sections of commented out code that contain work in progress, or previous versions of methods.
+- The code is clean and well written, but it doesn't reflect the user's vocabulary:
 
 > patient.setShotType(ShotTypes.TYPE_FLU);
```
Comment (joshprice) [inline _capability/4-development/index.md:32]: not a clear example in isolation, remove

File: _capability/4-development/index.md [hunk 7]
```diff
@@ -67,9 +94,6 @@ Resources:
 > nurse.administerFluVaccine(patient, vaccine);
```
Comment (joshprice) [inline _capability/4-development/index.md:94]: use triple backticks
`nurse.administerVaccine(patient, vaccine)`

File: _capability/6-quality/index.md [hunk 8]
```diff
@@ -20,27 +19,50 @@ Testing ensures that:
...
+The team have developed the confidence to experiment and innovate to help deliver the user needs. Code is continually refactored to remove technical debt, greatly accelerating the rate of delivering new features.
 
+### Indications that you're operating at level 2
 
-### Level 2
- - Code analysis tools are used to identify code coverage, which is 100% for the majority the code base
- - Automated integration and acceptance testing tools are used to ensure that the overarching system works as expected
- -  Dependency injection and mocking frameworks are used to isolate the components under-test from the rest of the system, allowing the components to be tested for a variety of circumstances independently of the dependencies, such as a database.
- - The team have a 'zero exceptions in production' mantra.
+- Code analysis tools are used to identify code coverage, which is near 100% for the majority the code base, and particularly hairy sections of the code have several tests to ensure the edge cases work.
```
Comment (joshprice) [inline _capability/6-quality/index.md]: 100% code coverage is often a bogus metric, I've seen codebases with 100% coverage but no actual assertions

File: _capability/7-feedback/index.md [hunk 9]
```diff
@@ -4,37 +4,48 @@ section: 7-feedback
...
+
+# Level 2
+
+Ongoing interaction between the team and users allows the team to really focus on where the pain points are, and what they can do to help.
+
+### Indications that you're operating at level 2
+
+- Users are able to request modifications and enhancements to the system, and the community vote and prioritise the things that are important to them.
+- The system provides anonymised information about how the users actually use the system, allowing the developers to see if features are used as expected, and what optimisations can be made for a better user experience
+- The system is able to identify when bugs occur, and automatically raise support tickets, alleviating the user from needing to describe what they were doing when the issue occurred
```
Comment (joshprice) [inline _capability/7-feedback/index.md:51]: - put developers on first line support or getting them onsite with customers to build empathy
